### ISSUE TITLE
letsencrypt-auto --apache fails with "AttributeError: 'NoneType' object has no attribute 'strip'"

### ISSUE THREAD

**Issue by FrancisRussell** [certbot/certbot#1738]
I am running Debian testing amd64 with apache2. Having cloned the git repo (currently 8bfda79fd663884e565a8c278a4fefd6c6cd5474), and having run './letsencrypt-auto --help all' which led to a number of dependencies being installed, running 'letsencrypt-auto --apache' fails as follows:

```
fpr@xen:~/letsencrypt$ ./letsencrypt-auto --apache
Updating letsencrypt and virtual environment dependencies.......
Running with virtualenv: sudo /home/fpr/.local/share/letsencrypt/bin/letsencrypt --apache
An unexpected error occurred:
AttributeError: 'NoneType' object has no attribute 'strip'
Please see the logfiles in /var/log/letsencrypt for more details.
```

[The log is here.](https://github.com/letsencrypt/letsencrypt/files/52035/letsencrypt.log.txt)

**Comment by takuya-o** [comment]
I met same error even if I used "--standalone" options under Debian testing amd64.

**Comment by 1stone** [comment]
Same here with commit 0c704fa and Debian Stretch (amd64).

**Comment by gdudek** [comment]
Same here on Ubuntu 12.04 letsencrypt version: 0.4.0

**Comment by bmw** [comment]
While the bug is manifesting itself in a different way, this may have been fixed by #2511 which solved a similar problem. That fix is in `master` and will be in the upcoming client release in the next week or so.

**Comment by joohoi** [comment]
This is an very old issue and a lot has been changed in certbot-apache since then. I'm very confident that this issue has been solved. If this is not the case, feel free to comment here and we'll reopen the issue.
